02.general/09.license-key/docs.en.md

```diff
@@ -16,13 +16,14 @@ visible: true
 <a name="key"></a>
 #### What is AdGuard license key?
 
-License key is a unique symbol sequence that activates AdGuard. License key is required for AdGuard’s work after trial period's expiration. There are few different types of licenses available:
+License key is a unique symbol sequence that activates AdGuard. License key is required for AdGuard’s work after trial period's expiration. There are two types of licenses available:
 
-1. **Standard** - allows you to activate AdGuard on Windows PC or Mac.
-2. **Mobile** - allows you to activate AdGuard on your Mobile device.
-2. **Premium** - mobile and standard in a single license. Allows you to activate both mobile device and computer with the same license key.
-4. **Bonus License** - is given for winning any contests or prize draws. Equivalent to standard license.
-5. **Beta-test** - special type of license for AdGuard beta-testers. Program activated with this type of license is automatically updated to the latest available beta.   
+1. **Personal** - This license key allows you to activate AdGuard on **any three** devices.
+
+2. **Family** - This license key allows you to activate AdGuard on **any nine** devices.
+
+**Please notice**: all types of licenses are universal. You can activate a license for any of the provided operating systems.
```

Comment:
`>**Please notice**: all types of licenses are universal. You can use any license to activate AdGuard on any of the operating systems — **Windows**, **Mac**, **Android**, or **iOS**.`